system-addon/content-src/styles/activity-stream-windows.scss

```diff
@@ -3,6 +3,5 @@
 $os-infopanel-arrow-height: 10px;
 $os-infopanel-arrow-offset-end: 6px;
 $os-infopanel-arrow-width: 20px;
-$os-search-focus-shadow-radius: 1px;
```

Comment:
Is this removal actually desired? It regresses #3745 which said to have 1px for windows.

Reply:
I asked Amy explicitly about it. I'll double confirm.

Reply:
the UX powers say we can get rid of the windows specific special case 🎉